### PR TITLE
changes for 5414 & 5415

### PR DESCRIPTION
**Pull request by MadhviAtroliya**
### :nut_and_bolt: Description: What code changed, and why?
Data feed page changed 
1. Disclaimer text added
2. New gif added to empty data-feed page
3. Create  data feed button changed to New Integration

### :chains: Related Resources
- https://github.com/chef/automate/issues/5414
- https://github.com/chef/automate/issues/5415

### :+1: Definition of Done
On Data feed page
1. 
image
changed to 
image
2. Added a button to add new Integration.

Automate data-feed page should look like the image below.
Screenshot 2021-08-18 at 3 41 38 PM

### :athletic_shoe: How to Build and Test the Change
1. Inside the hab studio
```
 rebuild component/automate-ui 
```
2. Once UI is up:
 Go to settings > click feat on tab to enable feature flag (on Chef Automate Data Feed option) and reload page - Data feed option should come below Notification tab
3. Click on data-feed 
4. Empty data-feed should look like the image.
Screenshot 2021-08-25 at 4 12 52 PM

To test:
1. Go to e2e/cypress folder in automate
2. run npm run cypress:local

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
Screenshot 2021-08-25 at 4 12 52 PM

### CODE REVIEW

File: e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        cy.get('chef-button').contains('Close').click();
+        cy.reload();
+        cy.contains('know').click();
+        cy.contains('Data Feeds').click();
+        cy.get('[data-cy=new-integration]').should('exist');
+      });
+    });
+
+
+});
```
Comment (vinay033) [inline e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts:41]: add EOL

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.scss [hunk 2]
```diff
@@ -7,23 +7,29 @@
 
 .filter-button chef-button {
   margin-left: 0;
+
```
Comment (vinay033) [inline components/automate-ui/src/app/pages/data-feed/data-feed.component.scss:10]: please delete these unused lines.

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.scss [hunk 3]
```diff
@@ -7,23 +7,29 @@
...
+  display: flex;
+  float: right;
+  align-items: center;
+  margin-top: -360px;
 }
 
 chef-icon#sort-desc {
   color: $chef-primary-dark;
 
+
```
Comment (vinay033) [inline components/automate-ui/src/app/pages/data-feed/data-feed.component.scss:24]: same here

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.html [hunk 4]
```diff
@@ -26,11 +27,18 @@
...
         <div *ngIf="(sortedDestinations$ | async)?.length === 0" class="empty-state">
-          <p>Create a data feed to get started!</p>
+          <img alt="No preview" src="/assets/img/no_preview.gif" />
+          <p>No Data Feeds Preview Available</p>
         </div>
         <div [ngClass]="(sortedDestinations$ | async)?.length === 0 ? 'empty-state' : ''">
           <!-- <chef-button primary [routerLink]="['/settings/data-feed/form']">Create Data Feed</chef-button> -->
-          <chef-button primary (click)="openCreateModal()">Create Data Feed</chef-button>
+          <chef-button primary (click)="openCreateModal()">
+            <svg aria-hidden="true" focusable="false" data-prefix="fal" data-icon="plus-circle" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"
```
Comment (vinay033) [inline components/automate-ui/src/app/pages/data-feed/data-feed.component.html:36]: for this you can do like

```
.dex-btn-icon--saml {
  background-color: #84B6EF; 
  background-image: url(../static/img/saml-icon.svg);
}
```
find this saml-icon.svg and check how we can do using svg file.

**Comment by vivek-yadav** [comment]
@MadhviAtroliya Please add the related issues as dependency to this PR.

Also, please go through the check list and make them checked if done.

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.html (hunk 4 above)
Comment (MadhviAtroliya) [reply components/automate-ui/src/app/pages/data-feed/data-feed.component.html]: Added svg file

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.scss (hunk 3 above)
Comment (MadhviAtroliya) [reply components/automate-ui/src/app/pages/data-feed/data-feed.component.scss]: Removed

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.scss (hunk 2 above)
Comment (MadhviAtroliya) [reply components/automate-ui/src/app/pages/data-feed/data-feed.component.scss]: Removed

File: e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts (hunk 1 above)
Comment (MadhviAtroliya) [reply e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts]: Added

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.scss [hunk 5]
```diff
@@ -5,51 +5,39 @@
...
 
 .actions {
   padding: 8px 1em;
 
   chef-button[icon] {
-    min-height: 32px;
     font-size: 18px;
     line-height: 32px;
-
+
```
Comment (vinay033) [inline components/automate-ui/src/app/pages/data-feed/data-feed.component.scss:21]: check `npm lint`
in `components/automate-ui` directory

File: e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
+describe('chef datafeed', () => {
+const now  = Cypress.moment().format('MMDDYYhhmmss');
```
Comment (vinay033) [inline e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts:2]: also, please take care of the indentation 😁

**Review by vinay033 — APPROVED** [review]
nice work, look good to me 🎉
Thanks @MadhviAtroliya

**Review by vivekshankar1 — APPROVED** [review]
Thank you, it looks good

File: e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts [hunk 7]
```diff
@@ -0,0 +1,39 @@
+describe('chef datafeed', () => {
+  const now  = Cypress.moment().format('MMDDYYhhmmss');
+  const cypressPrefix = 'infra';
+  const serverName = `${cypressPrefix} server ${now}`;
+  const generatedServerID = serverName.split(' ').join('-');
+  const customServerID = `${cypressPrefix}-custom-id-${now}`;
+  const serverFQDN = 'chef-server-1617089723092818000.com';
+  const serverIP = '176.119.50.159';
```
Comment (vivekshankar1) [inline e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts:8]: we dont need these constants

Comment (MadhviAtroliya) [reply e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts]: Removed

Comment (vivekshankar1) [reply e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts]: Thanks madhavi it looks fine now

File: components/automate-ui/src/app/pages/data-feed/data-feed.component.scss (hunk 5 above)
Comment (MadhviAtroliya) [reply components/automate-ui/src/app/pages/data-feed/data-feed.component.scss]: Done

File: e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts (hunk 6 above)
Comment (MadhviAtroliya) [reply e2e/cypress/integration/ui/infra-proxy/chef-datafeed.spec.ts]: Done

**Review by shaik80 — CHANGES_REQUESTED** [review]
The disclaimer section should have some more blank space at the top, please make sure matches the Figma design

**Review by shivangi-singh1 — APPROVED** [review]

**Review by vivek-yadav — APPROVED** [review]

**Review by shaik80 — APPROVED** [review]

**Pull request merged**